VenueNextSDK-Demo/AppDelegate.swift

```diff
@@ -7,42 +7,69 @@ import VNOrderUI
 import VNWalletUI
 import PresenceSDK
 
+class CustomWalletTheme: VNWalletBaseTheme {
+    override var accent: UIColor {
+        return navigationBarBackground
+    }
+
+    override var secondaryAccent: UIColor {
+        return UIColor(hexString: "FF0000")
```

Comment:
im assuming this working without the #

Reply:
Yeah, it works with or without the hash mark.